tests/testthat/_snaps/skip.md

```diff
@@ -88,3 +88,11 @@
 
     Reason: On Windows i386
 
+# skip_unless_r works as expected
+
+    Reason: R version requirement not satisfied: 4.5.0 >= 999.999.999
```

Comment:
I wonder if this would be better as "R version (4.5.0) does not satisfy requirement (>= 999.999.999)"?

Reply:
True, it's not strictly clear whether LHS or RHS is your current version.

What about "Current R version (...) does not satisfy..."?